pages/linux/xmount.md

```diff
@@ -0,0 +1,17 @@
+# xmount
+
+> Convert on-the-fly between multiple input and output hard disk image types with optional write cache support.
+> Creates a virtual file system using FUSE (Filesystem in Userspace) that contains a virtual representation of the input image.
+> More information: <https://www.pinguin.lu/xmount>.
+
+- Mount a RAW image file into a DMG container file:
```

Comment:
```suggestion
- Mount a `.raw` image file into a DMG container file:
```
I know this breaks the uniformity, but I think otherwise it can be read as `mount a raw image file` too easily.